src/seeds/seed.ts

```diff
@@ -1,10 +1,10 @@
 import { AppDataSource } from '../data-source';
-import { createFakerUser } from './factory';
+import { createFakerUsers } from './factory';
 
 //executa o DataSource antes de executar a função que cria os usuários
 async function seed() {
   await AppDataSource.initialize();
-  await createFakerUser(50);
+  await createFakerUsers(50);
```

Comment:
nit: quando vc for renomear qqr coisa que ja esta sendo usada em outros arquivos, vc pode renomear apertando F2. Isso vai fazer tdas as referencias atualizarem tb, ai vc nao precisa ficar caçando no codigo onde mudar